src/feature/NavigationBar/NavigationBar.jsx

```diff
@@ -0,0 +1,48 @@
+import { ROUTE } from "util/constant";
+import { Cta } from "../../ui/Cta";
+import { Profile } from "../../ui/Profile";
+import { LOGO_IMAGE, TEXT } from "./constant";
+import "./NavigationBar.css";
+import { useLocation } from "react-router-dom";
+import { useEffect, useState } from "react";
+
+export const NavigationBar = ({ userEmail, userImgSource }) => {
+  const [className, setClassName] = useState("NavigationBar");
+  const Location = useLocation();
+  const LocationPath = Location.pathname;
+  console.log(LocationPath);
+
+  const handleClassName = () => {
+    LocationPath === "/shared" || LocationPath === "/folder"
+      ? setClassName("NavigationBar-static")
+      : setClassName("NavigationBar");
+    console.log(className);
+  };
+
+  useEffect(() => {
+    handleClassName();
+  }, []);
```

Comment:
P1:
state로 하지 않아도 될 것 같네요.
그리고 요구사항만 봤을때는 폴더페이지에서만 상단 네비게이션바가 고정되지 않는것으로 보이는데 확인 부탁드려요.

```suggestion
  const Location = useLocation();
  const LocationPath = Location.pathname;
  const className = LocationPath === "/shared" || LocationPath === "/folder"
      ? "NavigationBar-static"
      : "NavigationBar";
```